core/src/main/java/com/datastax/oss/driver/internal/core/metadata/token/SimpleReplicationStrategy.java

```diff
@@ -30,22 +30,26 @@
 @ThreadSafe
 class SimpleReplicationStrategy implements ReplicationStrategy {
 
-  private final int replicationFactor;
+  private final ReplicationFactor replicationFactor;
 
   SimpleReplicationStrategy(Map<String, String> replicationConfig) {
     this(extractReplicationFactor(replicationConfig));
   }
 
   @VisibleForTesting
   SimpleReplicationStrategy(int replicationFactor) {
+    this(ReplicationFactor.fullOnly(replicationFactor));
+  }
```

Comment:
Rather than preserving this constructor, we can have unit tests create a replication factor explicitly and call `SimpleReplicationStrategy(ReplicationFactor)`.

Then maybe we can parameterize them to assert that a non-zero `transientReplicas` produces the same results.